client/lib/interval/README.md

```diff
@@ -1,43 +1,41 @@
 # Interval
 
-An interface into a global timer coalescing interval runner.
+A React-oriented, declarative interval runner.
 
-_**An interface**_ because this component handles registering and un-registering the given `onTick` action with the global runner.
-
-_**timer coalescing**_ because the global runner will run at the same time all of the actions registered for a given interval period. For example, if four actions are registered for running once per minute, they will all run at one point in time every minute instead of having four different run times, each time repeating every minute. This is better for things like battery life because it allows for longer and deeper periods of sleep between actions.
+This component can be used to easily trigger a polling, looping, or interval action in the background. Such usages could include polling an API endpoint for updates, sweeping over user input such as in the post editor at intervals, or updating an on-screen timer.
 
-_**interval runner**_ because the global runner will execute the given action every interval on the interval.
+It only requires two inputs: a callback to perform and an interval between which runs of the action should occur. The action is simply a function and the interval is a timeout in ms.
+Notice how this interface closely matches `setInterval`.
 
-This component can be used to easily trigger a polling, looping, or interval action in the background. Such usages could include polling an API endpoint for updates, sweeping over user input such as in the post editor at intervals, or updating an on-screen timer.
+## Usage
 
-It only requires two inputs: an action to perform and an interval between which runs of the action should occur. The action is simply a function and the interval is a named constant representing the interval period. These interval periods are intentionally limited to prevent sprawl of timers.
+### `useInterval`
 
-The action will only be executed as long as the React component is mounted, as the component un-registers the action on unmount. Additionally, the default behavior is to stop executing the action when the browser document is hidden, though this can be overwritten. "Hidden" means that another browser tab is selected or the browser is minimized.
+```tsx
+import { useInterval, EVERY_MINUTE } from 'react';
 
-Wrapped components will be transferred all additional props not consumed by the `<Interval />` itself.
+function Counter() {
+	let [ count, setCount ] = useState( 0 );
 
-## Usage
+	useInterval( () => {
+		setCount( count + 1 );
+	}, EVERY_MINUE );
 
-```jsx
-import Interval, { EVERY_FIVE_SECONDS, EVERY_MINUTE } from 'lib/interval';
+	return <h1>Minutes: { count }</h1>;
+}
+```
 
-<Interval onTick={ doSomething } period={ EVERY_FIVE_SECONDS } />
+### `<Interval />`
 
-// Wrapping a component
```

Comment:
This was never used and wouldn't work as far as I can tell. `Interval` always renders `null` 🤷‍♂

Reply:
See above comment: this _was_ used and _did_ work before the component was rewritten in https://github.com/Automattic/wp-calypso/commit/d68eef66197e571fb0fdf1561e5ee9cef7584479 and apparently the README was overlooked